atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -746,17 +745,14 @@ private static LockAndTimestampServices createRawServicesFromTimeLock(
                 .withMigrator(migrator);
     }
 
+    //  what's the deal with all these clients
     private static Supplier<ServerListConfig> getServerListConfigSupplierForTimeLock(
             AtlasDbConfig config,
             Supplier<AtlasDbRuntimeConfig> runtimeConfigSupplier) {
         Preconditions.checkState(!remoteTimestampAndLockOrLeaderBlocksPresent(config),
                 "Cannot create raw services from timelock with another source of timestamps/locks configured!");
-        TimeLockClientConfig clientConfig = config.timelock().orElse(ImmutableTimeLockClientConfig.builder().build());
-        String resolvedClient = OptionalResolver.resolve(clientConfig.client(), config.namespace());
-        return () -> ServerListConfigs.parseInstallAndRuntimeConfigs(
-                clientConfig,
-                () -> runtimeConfigSupplier.get().timelockRuntime(),
-                resolvedClient);
+        return () -> ServerListConfigs.addNamespaceToServerAddresses(
+                runtimeConfigSupplier.get().timelockRuntime().serversList(), config.namespace().get());
     }
```

Comment:
You previously could specify a TimeLock client specifically (that was different from your namespace). That was done at the time namespaces were introduced to avoid a user-break in terms of config. Now that you're removing the timelock client configuration option, the following looks OK.

Reply:
Yeah the corroboration check will catch this.

When we do this release we could call out that if your client was _not the same_ as your KVS keyspace or namespace, you need to contact the AtlasDB team. A special manual migration task may be needed when upgrading AtlasDB to a version that has this check.

Reply:
Where should I add that announcment Should I add that in the release note?